server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -266,27 +266,29 @@ export function configureWebSocketServices(
 			}, mSecUntilExpiration);
 		}
 
-        async function connectDocument(message: IConnect): Promise<IConnectedClient> {
```

Comment:
I think this file contains only space changes. If so, I'd recommend making separate PR and mention in PR that it's no-change. Makes it so easier to track history a year from now.